Makefile

```diff
@@ -164,6 +164,16 @@ talos-gce: installer
 	@tar -C $(PWD)/build -Sczf $(PWD)/build/$@.tar.gz disk.raw
 	@rm $(PWD)/build/disk.raw
 
+.PHONY: talos-aws
+talos-aws: installer
+	@docker run \
+		--rm \
+		-i \
+		-e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) \
+		-e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY) \
+		-e AWS_DEFAULT_REGION=$(AWS_DEFAULT_REGION) \
+		autonomy/installer:$(TAG) ami -var regions=${AWS_REGIONS}
```

Comment:
Are we going to need to update drone for this or not yet?

Reply:
I'm thinking not yet. This at least allows us to run it manually on a release. Kinda like what we're doing with GCE stuff.

Reply:
But if you feel we should just do it now, then I'm ok with that too.